lms/templates/courseware/course_navigation.html

```diff
@@ -21,6 +22,10 @@
     settings.FEATURES.get('ENABLE_SPECIAL_EXAMS', False) and
     (course.enable_proctored_exams or course.enable_timed_exams)
     )
+
+masquerading_as_student = False
```

Comment:
Nit: Why need 3 lines when you can do more in just one line? 

`masquerading_as_student = is_masquerading_as_student(request.user, course.id)`